frame_metadata/TypeVisitor.ts

```diff
@@ -1,21 +1,21 @@
 import * as U from "../util/mod.ts";
 import * as M from "./Metadata.ts";
 
-export type UnknownByTypeDefKind = { [Tag in M.TypeKind]: unknown };
+export type UnknownByTypeDefKind = { [Tag in M.TyType]: unknown };
 
 export type TypeVisitor<
-  Tag extends M.TypeKind,
+  Tag extends M.TyType,
```

Comment:
Should the name of the type param also change? It is technically still a tag.

Reply:
... still, might be better as `TyType`

Reply:
Yep, it should